apps/server-asset-sg/src/features/contacts/contacts.controller.ts

```diff
@@ -10,18 +10,16 @@ import {
   Put,
   ValidationPipe,
 } from '@nestjs/common';
-
-import { RequireRole } from '@/core/decorators/require-role.decorator';
+import { Authorize } from '@/core/decorators/authorize.decorator';
 import { Contact, ContactDataBoundary, ContactId } from '@/features/contacts/contact.model';
 import { ContactRepo } from '@/features/contacts/contact.repo';
-import { Role } from '@/features/users/user.model';
 
 @Controller('/contacts')
 export class ContactsController {
   constructor(private readonly contactRepo: ContactRepo) {}
 
   @Post('/')
-  @RequireRole(Role.Editor)
+  @Authorize.Admin()
```

Comment:
why can only admin add new contacts? sholdn't editors also be allowed to do this?